src/vmm/src/seccomp_filters/mod.rs

```diff
@@ -2,18 +2,9 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
This is a 'one file module'. Can be renamed to `seccomp_filters.rs`.

Also nit: because this file is already heavily modified, maybe replace the `impl fmt::Display for FilterError` with `thiserror`?

Reply:
Good point, might as well :D

Reply:
Mh, yeah a lot more can be moved to the binary crate, true - but moving `get_empty_filters` to `test_utils` doesn't "work" in the sense that then we'd have production code calling into a `test_utils` module, which is aesthetically unpleasing.

Reply:
is the `get_empty_filters` used by production code?

Reply:
When starting firecracker, theres a CLI option to disable seccomp, which causes `get_empty_filters` to be used